### ISSUE TITLE
Terraform v0.12.0-beta1 compatibility

### ISSUE THREAD

**Issue by danielposton** [adamdecaf/terraform-provider-namecheap#13]
Thank you for making a namecheap provider!  I was wondering if it's on the radar to make a version that is compatible with the new v0.12.0-beta1 version?

**Comment by adamdecaf** [comment]
@danielposton Yea. I haven't looked over what exactly has changed, but I'm aware an upgrade is needed. I'll try and read over what all is changing this weekend.

**Comment by adamdecaf** [comment]
@danielposton I created https://github.com/adamdecaf/terraform-provider-namecheap/pull/14 which worked for me locally on a DNS record. Can you try out the branch? Just run `make` to install the binary on your system in `$GOPATH/bin`. 

https://github.com/adamdecaf/terraform-provider-namecheap/tree/terraform0.12

**Comment by NicoleYson** [comment]
I tried to do this myself, because all namecheap resources are stuck on `module.test.namecheap_record.subdomain.1: Still creating... (40m31s elapsed)`
 for me 

Upon a `make` from that `terraform0.12` branch - 
```
fatal: unable to access 'https://gitlab.howett.net/go/plist.git/': SSL certificate problem: certificate has expired
```
Are there any alternatives that can be used, you think?

Version for context:
```
terraform --version
Terraform v0.11.13
```

**Comment by adamdecaf** [comment]
`gitlab.howett.net` looks to be using a Let's Encrypt certificate chain. Is your machine trusting that root? 

https://api.moov.io/ is another site (that I host) which uses Let's Encrypt. Can you load that?

**Comment by OGProgrammer** [comment]
Maybe he is behind a proxy or something. Also I noticed `export TF_LOG=trace` before running `terraform apply` might give you more info here.

**Comment by adamdecaf** [comment]
FYI the master branch is updated with 0.12.0-rc1 after Hashicorp released it.

**Comment by adamdecaf** [comment]
1.3.0 has been released with Terraform 0.12 support.
